src/index.js

```diff
@@ -0,0 +1,150 @@
+let allCityDetails
+
+/**
+ * This function will change the icons for particular hour based on temperature value
+ * @param {string} temp current temperature
+ * @param {number} i increment value for accessing each classes iteretively
+ */
+function changeIconImage (temp, i) {
```

Comment:
The function naming could be little more relevant to our use case.
Maybe like changeSelectedCityIcon